platform-sdk/swirlds-benchmarks/settings.txt

```diff
@@ -9,5 +9,6 @@ benchmark.csvAppend,                           true
 virtualMap.flushInterval,                      20
 virtualMap.preferredFlushQueueSize,            1
 virtualMap.flushThrottleStepSize,              500ms
+virtualMap.reconnectMode,                      pullTwoPhasePessimistic
```

Comment:
Are you sure you want this to be the default for benchmarks going forward? It may be forgotten, and then the benchmarks make produce confusing results. I suggest to use the same default value here that is used in the platform by default currently.

Reply:
I can hardly believe a person running these benchmarks can "forget" about the mode and be confused about unexpected results. Nevermind, I can revert this change

Reply:
I would.

I'll say more: I wouldn't even think about opening and checking this file in the first place, lol. :) I think it's fair to assume that by default the benchmark should be testing what the platform is doing by default. Any customizations should be passed via command-line parameters or by local temporary modifications in your local repository.